src/olmo_core/train/callbacks/wandb.py

```diff
@@ -151,11 +151,11 @@ def check_if_canceled(self):
             try:
                 # NOTE: need to re-initialize the API client every time, otherwise
                 # I guess it return cached run data.
-                api = self.wandb.Api(api_key=os.environ[WANDB_API_KEY_ENV_VAR])
+                api = self.wandb.Api(api_key=os.environ[WANDB_API_KEY_ENV_VAR], timeout=5)
```

Comment:
Unrelated change?

Reply:
Sort of related. Good to have a timeout on this.